### PR TITLE
Gherkin scenarios for the Multi-cluster Overview page

### PR DESCRIPTION
**Pull request by matejnesuta**
Related to #6477.

Drafting this for now, because I might come up with some more scenarios later.

### CODE REVIEW

File: frontend/cypress/integration/featureFiles/overview.feature [hunk 1]
```diff
@@ -137,10 +137,64 @@ Feature: Kiali Overview page
     Then user sees the memory chart
     And user sees the cpu chart
 
+  @overview-page
+  @multi-cluster
+  @skip
+  Scenario: The badge for local cluster should be visible
```
Comment (matejnesuta) [inline frontend/cypress/integration/featureFiles/overview.feature:143]: This is just a simple badge check. After seeing this PR (https://github.com/kiali/kiali/pull/6019), I think the local cluster name could be obtained somewhere from the configs, but I am not sure where to look for it exactly.

**Comment by matejnesuta** [comment]
There is also #6092, which could be tested on the Overview page as well, but now the mTLS is not enabled afaik.

**Review by ScriptingShrimp — DISMISSED** [review]
LGTM

File: frontend/cypress/integration/featureFiles/overview.feature [hunk 2]
```diff
@@ -120,9 +120,58 @@ Feature: Kiali Overview page
...
     And user sees the "bookinfo" namespace card in cluster "west"
-    And Istio config should not be available for the "west" "bookinfo"
+    And Istio config should not be available for the "west" "bookinfo" 
+    And health should be different for "east" and "west" "bookinfo"
+
+  @multi-cluster
+  @skip
+  Scenario: The healthy status of a logical mesh application is reported in the overview of a remote cluster namespace
+    Given a healthy application in the "west" cluster
+    When I fetch the overview of the "west" cluster
```
Comment (hhovsepy) [inline frontend/cypress/integration/featureFiles/overview.feature:158]: it seems the "$cluster" param does not exist for these functions, can you add t in function? or remove "west" from the call.

**Review by hhovsepy — APPROVED** [review]
lgtm

**Pull request merged**